From e33716c448c3e907efb819ec25e88549895a3ae7 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 2 Jun 2023 11:24:59 +0200
Subject: [PATCH 0218/1141] arm64: defconfig: enable FMC2 support

This patch adds FMC2 EBI and FMC2 NAND controllers support
used by STM32MP SOCs.

Change-Id: I70d1374974823239b137e9242045564c92988216
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310423
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 3302967bd2e9..dd2bb2c62847 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -12,6 +12,7 @@ CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_MFD_STM32_TIMERS=m
 CONFIG_MFD_WM8994=m
+CONFIG_MTD_NAND_STM32_FMC2=y
 CONFIG_NET_ACT_POLICE=m
 CONFIG_NET_ACT_SKBEDIT=m
 CONFIG_NET_ACT_VLAN=m
@@ -39,6 +40,7 @@ CONFIG_SPI_STM32=m
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_DMA3=y
+CONFIG_STM32_FMC2_EBI=y
 CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STM32_MDF_ADC=m
 CONFIG_STM32_MDF_CORE=m
-- 
2.39.2

